transports/index.md

```diff
@@ -1,72 +1,38 @@
 ---
 title: Transports
 summary: An overview of the transports available in NServiceBus
-reviewed: 2018-08-26
+reviewed: 2018-11-27
 redirects:
  - nservicebus/nservicebus-and-websphere-sonic
  - nservicebus/transports
 related:
  - samples/custom-transport
 ---
 
-NServiceBus is built on top of existing queuing technologies. In NServiceBus, the choice of queuing technology is referred to as a "Transport".
+NServiceBus contains an abstraction for underlying queueing technologies. An implementation of that abstraction for a given queueing technology is known as a "transport".
 
+The transport abstraction enables business to build systems with the Particular Service Platform using existing queueing technologies. Sometimes the choice of queueing technology may be constrained be constrained by the business. At other times the best queueing technology can be selected according to requirements. The transport abstraction allows more time to be spent delivering features and less time to be spent dealing with the details of a specific queueing technology.
```

Comment:
```suggestion
The transport abstraction enables businesses to build systems with the Particular Service Platform using existing queueing technologies. Sometimes the choice of queueing technology may be constrained by the business. At other times the best queueing technology can be selected according to a different set of requirements. The transport abstraction allows teams to spend more time delivering business features than dealing with specific queueing technology details.
```

The original version doesn't sound right to me. I'm not sure how to tweak it.

Reply:
Changed to

> The transport abstraction enables businesses to build systems with the Particular Service Platform using existing queueing technologies. Sometimes the choice of queueing technology may be constrained by the business. At other times the best queueing technology can be selected according to requirements. The transport abstraction allows teams to spend more time delivering business features and less time dealing with the details of a specific queueing technology.